src/pyshark/capture/capture.py

```diff
@@ -344,6 +357,17 @@ async def _get_tshark_process(self, packet_count=None, stdin=None):
         parameters = [self._get_tshark_path(), "-l", "-n", "-T", output_type] + \
             self.get_parameters(packet_count=packet_count) + output_parameters
 
+        if not tshark_supports_protocol(self._display_filter):
```

Comment:
TBH this seems like overkill to check it every time we get the process, since it'll only fail very rarely. Maybe we can check it on a crash to explain it?